keyboards/hp69/keymaps/default/rules.mk

```diff
@@ -0,0 +1 @@
+AUDIO_ENABLE = yes
```

Comment:
Is audio optional, if not this can be moved to the keyboard level.

Reply:
I'd like to keep it on by default, thanks